src/include/distributed/worker_transaction.h

```diff
@@ -24,7 +24,8 @@ typedef enum TargetWorkerSet
 {
 	WORKERS_WITH_METADATA,
 	OTHER_WORKERS,
```

Comment:
I believe OTHER_WORKERS is not used at all. Can we just remove it then?

Reply:
https://github.com/citusdata/citus/blob/master/src/backend/distributed/transaction/worker_transaction.c#L213 seems to be used

Reply:
I meant it's never set to that value, so that check can also be removed then.